### PR TITLE
Fix for #4697

### PR DESCRIPTION
**Pull request by fabsx00**

### CODE REVIEW

**Comment by ret2libc** [comment]
Hi! Thanks for the contribution. Please read https://github.com/radare/radare2/blob/master/CONTRIBUTING.md#coding-style-guidelines and try to fix some issues in your patch (there are some missing spaces around braces and `if`/`else` etc.)

File: libr/anal/xrefs.c [hunk 1]
```diff
@@ -29,12 +29,23 @@ static void XREFKEY(char * const key, const size_t key_len,
...
 R_API int r_anal_xrefs_load(RAnal *anal, const char *prjfile) {
-	char *path, *db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
+		char *path, *db;
 	ut8 found = 0;
 	SdbListIter *it;
 	SdbNs *ns;
-	if (!db) return false;
-	path = r_str_home (db);
+
+	if(!prjfile || strlen(prjfile) < 1) return false;
```
Comment (radare) [inline libr/anal/xrefs.c]: if strlen ()<1 it means strlen == 0, therefor the only thing you need to check here if the first char is not \0, strlen is an unnecessary slowdown

**Comment by radare** [comment]
please squash commits for proper review

**Comment by radare** [comment]
Fixed, squashed and merged by hand

**Pull request closed without merging**